src/odemis/gui/cont/acquisition.py

```diff
@@ -1500,8 +1509,19 @@ def on_acquisition(self, evt):
         self._show_status_icons(None)
         self._tab_panel.Layout()  # to put the gauge at the right place
 
-        # start acquisition + connect events to callback
-        self.acq_future = acqmng.acquire(self._tab_data_model.acquisitionStreams, self._main_data_model.settings_obs)
+        # fold the streams if possible
+        folds = tuple(acqmng.foldStreams(self._tab_data_model.acquisitionStreams))
+
+        if len(self._tab_data_model.acquisitionStreams) > len(folds):
+            self.lbl_acqestimate.SetLabel("EBIC and CL streams acquired simultaneously")
+            # start acquisition
+            self.acq_future = acqmng.acquire(folds, self._main_data_model.settings_obs)
+        else:
+            # start acquisition
+            self.acq_future = acqmng.acquire(self._tab_data_model.acquisitionStreams,
+                                             self._main_data_model.settings_obs)
```

Comment:
That's not what I meant. You can always use `acquire(folds)`, because either folds is the same as `acquisitionStreams`, or it's the right content.
```suggestion
        if len(self._tab_data_model.acquisitionStreams) > len(folds):
            self.lbl_acqestimate.SetLabel("EBIC and CL streams acquired simultaneously")
        # start acquisition
        self.acq_future = acqmng.acquire(folds, self._main_data_model.settings_obs)
```